node/src/effect.rs

```diff
@@ -1196,13 +1169,16 @@ impl<REv> EffectBuilder<REv> {
     }
 
     /// Passes a finalized proto-block to the block executor component to execute it.
```

Comment:
```suggestion
    /// Passes a finalized proto-block to the block executor component to execute it.
    /// The `deploys` must match the deploy and transfer hashes in the finalized block, in order.
```

Reply:
Fixed in 5f80f2f45340c67321a7adc3cc30dba8ed510b25